dev/TEAL.md

```diff
@@ -353,6 +400,8 @@ Asset fields include `AssetHolding` and `AssetParam` fields that are used in `as
 | `dup` | duplicate last value on stack |
 | `dup2` | duplicate two last values on stack: A, B -> A, B, A, B |
 | `dig n` | push the Nth value from the top of the stack. dig 0 is equivalent to dup |
+| `cover n` | remove top of stack, and place it deeper in the stack such that N elements are above it |
```

Comment:
what is the range of N
Can N be 0? In which case, does it do nothing?

Reply:
Correct.  It is useless, so we could outlaw it in case we think of something more useful for it to do.  But it doesn't seem worth it.